Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -395,18 +395,6 @@ - (void)distributeNoReleaseAvailable:(MSACDistribute *)distribute {
   [alert addAction:[UIAlertAction actionWithTitle:NSLocalizedString(@"OK", nil) style:UIAlertActionStyleDefault handler:nil]];
   [self.window.rootViewController presentViewController:alert animated:YES completion:nil];
 }
-
-- (void)distributeWillExitApp:(MSACDistribute *)distribute {
```

Comment:
Delegate should be still implemented, that's why it's here

Reply:
The idea is to move `sleep` to view and remove it from here.

Reply:
To reproduce do this:
- press Simulate App exit on the distribution page (should wait 7 sec and then close the app)
- update the app using standart distribution process (should close app for update without a delay)

Reply:
If we want to show an alert inside delegate, then we need to show another update before closing app. So for    **Simulate App exit** we will show another alert in code. It duplicates code a bit but works the necessary way.